frontend/css/main.less

```diff
@@ -283,13 +310,28 @@ pre code.hljs {
     > :not(:first-child) {
       color: @orange;
     }
+
     margin-right: 0.7em;
   }
 }
 
 // For cover art in alerts
-@alertThumbnailSize: 60px;
-.alert .alert-thumbnail {
+@alertThumbnailSize: 40px;
+
+.alert {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.alert-text {
+  font-size: 11px;
+  font-weight: bold;
+  margin-left: 9px;
+  color: #353070;
+}
+
+.alert-thumbnail {
```

Comment:
A few details here:

- we should be careful about overloading standard class names. in this case, ".alert" is also a bootstrap class with it own styles, so it would be wise to use something like `.brainzplayer-alert` to avoid confusion and style overlap
- I don't think the modifications to the text styles we necessary. I would remove all the `.alert-text` styles
- the margin between the thumbnail and the text is required however, which we can instead add to `.alert-thumbnail` as `margin-right: @alert-padding;` instead of ` margin-right: 0;`